### PR TITLE
[FEATURE] Dictionaries

### PR DESCRIPTION
**Pull request by tailaiwang**
1. Added all words in the English language to english.txt. [Source](https://github.com/dwyl/english-words)

### CODE REVIEW

**Review by tailaiwang — COMMENTED** [review]
Real simple - just two .txt files
1. phrases.txt includes some curated idioms and phrases from yours truly
2. english.txt contains the entire oxford english dictionary in txt form

File: phrasle/public/phrases.txt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+OUT OF THE LOOP
+ELEPHANT IN THE ROOM
+BARK UP THE WRONG TREE
+CAUGHT RED HANDED
+UNDERSTOOD THE ASSIGNMENT
+GREEN LIGHT
+COLD TURKEY
+NAIL IN THE COFFIN
+BETWEEN A ROCK AND A HARD PLACE
+PUSHING P
```
Comment (BraydenRoyston) [inline phrasle/public/phrases.txt:90]: P

Comment (tailaiwang) [reply phrasle/public/phrases.txt:90]: 🅿️

**Review by BraydenRoyston — APPROVED** [review]
What a stupendous display of technical ability. A job well done to all involved.

**Comment by ericdachen** [comment]
I concur with Mr. Royston’s commentary

**Review by ericdachen — APPROVED** [review]

**Pull request merged**